doc/quickstart.md

```diff
@@ -17,7 +17,8 @@
 | Mellanox MT28908 Family [ConnectX-6] | 15b3 | 101b |
 | Mellanox MT28908 Family [ConnectX-6 Dx] | 15b3 | 101d |
 | Mellanox MT42822 BlueField-2 integrated ConnectX-6 Dx | 15b3 | a2d6 |
-
+| Qlogic  QL45000 Series Gigabit Ethernet Controller | 1077 | 1664 |
```

Comment:
so are you adding two types of NICs ?

vendor ID : 1077 , device ID 1664
vendor ID: 1077 , device ID 1654 

according to the configMaps above , the PF (physical function) device ID is 1654, and the VF (Virtual function) device ID is 1664

if this is the case, then you need just one line here containing the PF information only.

Reply:
removed VF ID in readme